Introduce a new render_in_view_context test helper to render arbitrary logic in the view (#1339)
github/view_component: docs/api.md

assert_text("Hello, World!")
```

> If `component` is `nil`, `render_inline` will execute the block in the view context instead:

Reviewer: ```suggestion
If no `component` is provided, `render_inline` executes the block in the view context:
```

```ruby
render_inline do
  render(MyComponent.new)
end

assert_text("Hello, World!")
```

> 

Reviewer: I think these changes should be removed ❤️ 

The API docs are auto-generated anyways.

Author: Yeah... I thought the docs were generated and committed back to the repo, but nothing happened this time? Maybe I'm thinking of a different repo.
